eox_tagging/test/integration/test_api_integration.py

```diff
@@ -0,0 +1,173 @@
+"""
+Integration tests.
+
+This suite performs multiple http requests to guarantee
+that the eox-tagging rest API is behaving as expected on a live server.
+"""
+import json
+from os import environ
+from unittest import skipUnless
+
+import requests
+from django.test import SimpleTestCase
+from rest_framework import status
+
+
+@skipUnless(environ.get("TEST_INTEGRATION"), "integration")
+class TagTestCase(SimpleTestCase):  # pragma: no cover
+    """Test suite"""
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        Initialize data fixtures
+        """
+
+        with open("eox_tagging/test/integration/test_data.json") as file:
```

Comment:
will this file always be the same ?

Reply:
You are supposed to edit the file's content whenever you want to run the tests, but the file's name should always stay the same.

Reply:
that means that you have to create a new release of eox-tagging every time that you want to run a different test ??

Reply:
You don't have to. The idea is to have a Jenkins job, a script or some kind of automated system that setups that file and run `TEST_INTEGRATION=true make test-python`

Reply:
So jenkins has to install the plugin in editable mode and then modifies the file, doesn't it ?

Reply:
Rather, Jenkins clones the repo and modifies the file, and runs the test. There's no need to install the plugin (assuming obviously that is insalled on the server you want to test).

Reply:
did you consider to get those values from the environment variables or use the settings file and overwrite that with the environment values ?? imo both options are better than overwriting a file